pkg/openstack/cinder.go

```diff
@@ -41,6 +41,20 @@ func ReconcileCinder(ctx context.Context, instance *corev1beta1.OpenStackControl
 			//cinder.Spec.DatabaseInstance = instance.Name // name of MariaDB we create here
 			cinder.Spec.DatabaseInstance = "openstack" //FIXME: see above
 		}
+		// if already defined in the top-level crd, we don't override
```

Comment:
I am not sure if this comment reflects the code, or I miss-read it. isn't it that if you haven't specified it a specific volume list in the cinder template it will use the global?

Reply:
Yes, I'll clarify this comment in my next push: the design choice is that if you define `extravolumes` in the template section we don't merge them with the top-level defined extra volumes, hence that `if statement` is needed to entirely skip the part where we merged `cinder.Spec.ExtraMounts` with `instance.Spec.ExtraMounts`.
We can change the approach in a follow up, but for now let's try to move forward with this change.